index.html

```diff
@@ -0,0 +1,139 @@
+          </p>
+          <p>
+            There is a wide variety of wind instruments that can be seen around the world. Some of the highly renowned wind instruments are:
+          </p>
+          <ul>
+            <li>Flute</li>
+            <li>Oboe</li>
+            <li>Clarinet</li>
+            <li>Bassoon</li>
+            <li>Saxophone</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="string_instruments">
+        <header>String Instruments</header>
+        <article>
+          <code></code>
+          <p>
+            The instruments that fall in this category consist of strings that can be vibrated to produce different sounds with varying pitches. The pitch of the sound produced via these instruments is highly dependent on the length of the air column inside the instrument. The pitch also significantly depends upon the thickness of the strings used in the instrument.
+          </p>
+          <p>
+            Some examples of string instruments include:
+          </p>
+          <ul>
+            <li>Guitar</li>
+            <li>Violin</li>
+            <li>Viola</li>
+            <li>Double Bass</li>
+            <li>Cello</li>
+            <li>Harp</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="percution_instruments">
+        <header>Percution Instruments</header>
+        <article>
+          <code></code>
+          <p>
+            Percussion instruments are the musical instruments in which sound is produced by beating or scraping them using a beater, hand, or a similar instrument. These instruments are broadly divided into two classes:
+          </p>
+          <ol>
+            <li>
+              <strong>Pitched Percussion</strong> instruments are instruments that produce notes with an identifiable pitch.
+            </li>
+            <li>
+              <strong>Unpitched Percussion</strong> instruments are the ones that produce notes with a pitch that is not identifiable.
+            </li>
+          </ol>
+          <p>
+            These instruments are a bit tricky to play because it takes a lot of experience and practice to be able to hit the instrument with the right amount of intensity for a given tone. These instruments are widely used as they are quite melodic and make special sounds that add excitement and vibrance to the atmosphere. Some examples are:
+          </p>
+          <ul>
+            <li>Piano</li>
+            <li>Triangle</li>
+            <li>Tambourine</li>
+            <li>Chimes</li>
+            <li>Cymbals</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="electronic_instruments">
+        <header>Electronic Instruments</header>
+        <article>
+          <code></code>
```

Comment:
Could you please consider removing the empty tags

Reply:
- [ ] Kindly check my comments on line `47` above on the `why` implementation 👍